tests/network/primary_pod_network.go

```diff
@@ -223,12 +226,7 @@ func vmiWithMasqueradeBinding() *v1.VirtualMachineInstance {
 	return vmi
 }
 
-func newFedoraWithGuestAgentAndDefaultInterface(iface v1.Interface) (*v1.VirtualMachineInstance, error) {
-	networkData, err := libnet.CreateDefaultCloudInitNetworkData()
```

Comment:
Why passing the `networkData` is better than automatically create it inside the method?